src/Uno.UWP/UI/Input/GestureRecognizer.cs

```diff
@@ -68,24 +68,32 @@ public void ProcessDownEvent(PointerPoint value)
 			}
 		}
 
-		public void ProcessMoveEvents(IList<PointerPoint> value)
+		public void ProcessMoveEvents(IList<PointerPoint> value) => ProcessMoveEvents(value, true);
+
+		internal void ProcessMoveEvents(IList<PointerPoint> value, bool isRelevant)
 		{
-			foreach (var point in value)
+			if (isRelevant)
```

Comment:
This `isRelevant` concept is nice, but doesn't answer the question "relevant for what??"

Reply:
Relevant to be handled / propagated by the class ... I understand your point, but I'm not sure how to make it clearer 🤔

Reply:
`isRelevantForPropagation`

Reply:
I've renamed it to `isOverOrCaptured` and add a comment to explain that this name does reflect the complete complexity of this boolean.